### DISCUSSION TITLE
Імена персонажів

### DISCUSSION THREAD

**Discussion by IndiMops** [IndiMops/translate-Red-Deat-Redemption-2#1] [Ideas]
Імена це завжди важкий момент в кожному перекладі, а найкращий спосіб вирішити цю проблему це звісно спитати думки у вас друзі. Тому якщо у вас є пропозиції, щодо правки імен, які ми пропонуємо[[тут](https://github.com/IndiMops/translate-Red-Deat-Redemption-2/blob/main/name-characers.txt)]. Будемо раді почути від вас вашу думку!

**Comment by sspaceless** [comment]
https://github.com/IndiMops/translate-Red-Deat-Redemption-2/blob/c611e0822944550cca692d58606a41cad12f4a84/name-characers.txt#L8

Здається тут має бути `Пані` ;)

**Reply by IndiMops** [reply]
Дякую, що помітили. Даної помилки в текстах наче немає. Цей файл служить, як довідка для перекладачів та редакторів

**Comment by Keeach** [comment]
https://github.com/IndiMops/translate-Red-Deat-Redemption-2/blob/04c028fbad0b543d04368f027776276d9557f60d/name-characers.txt#L35

Я вважаю якось дивно звучить Хозія - мені здається що краще щоб було ближче до оригіналу, як це ім'я вимовляють інші герої гри - Хозея 😇

**Reply by IndiMops** [reply]
Звернемо увагу
